src/Link.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { Text } from 'react-native';
+import PropTypes from 'prop-types';
+
+import normalize from './helpers/normalize';
+import GalioTheme, { withGalio } from './theme';
+
+function Typography({
+    style,
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6,
+    p,
+    body,
+    small,
+    muted,
+    neutral,
+    size,
+    color,
+    bold,
+    italic,
+    center,
+    children,
+    styles,
+    theme,
+    ...rest
+  }) {
+    return (
+        <TouchableOpacity onPress={() => Linking.openURL(url)}>
```

Comment:
This could be an URL link or it could be just a way into another screen. In order to let the user choose whatever he wants to do with the `<Link />` component we'd have to do something like this: `onPress={() => onPress()}`.  The 2nd `onePress` is a prop which our component gets. So you'd have to integrate it in the props part of the function.